core/plugins/org.polarsys.capella.core.sirius.analysis/description/context.odesign

```diff
@@ -2522,6 +2514,11 @@
           <reusedTools xsi:type="tool:ToolGroup" href="common.odesign#//@ownedViewpoints[name='Common']/@ownedRepresentations[name='Class%20Diagram%20Blank']/@defaultLayer/@toolSections.1/@ownedTools[name='TitleBlock']"/>
           <reusedTools xsi:type="tool_1:ToolDescription" href="common.odesign#//@ownedViewpoints[name='Common']/@ownedRepresentations[name='Class%20Diagram%20Blank']/@defaultLayer/@toolSections.1/@ownedTools[name='title.blocks']"/>
         </toolSections>
+        <customization>

```

Comment:
a customization allows to make "bold label" on all conditional styles without having to do all combinatory.
i have added a customisation per diagram but we can create a global one on common.odesign for all breakdown. for instance, a new diagram "Common > Common Description" containing these common things, with no possibility to create such diagram.
idk, wdyt?